lib/src/realtime_channel.dart

```diff
@@ -78,11 +76,9 @@ enum ChannelResponse {
   }
 }
 
-enum RealtimeListenTypes {
-  postgresChanges,
-  broadcast,
-  presence;
+enum RealtimeListenTypes { postgresChanges, broadcast, presence }
 
+extension ToType on RealtimeListenTypes {
```

Comment:
Here as well

Reply:
Thanks for picking up on these! I think this one we can just hide it and not make it accessible to the users!